src/domain/valueObject/sslPairId.go

```diff
@@ -0,0 +1,48 @@
+package valueObject
+
+import (
+	"errors"
+	"regexp"
+
+	voHelper "github.com/goinfinite/os/src/domain/valueObject/helper"
+)
+
+const sslPairIdExpression = "^[a-fA-F0-9]{64}$"
+
+type SslPairId string
+
+func NewSslPairId(value interface{}) (sslPairId SslPairId, err error) {
+	stringValue, err := voHelper.InterfaceToString(value)
+	if err != nil {
+		return sslPairId, errors.New("SslPairIdMustBeString")
+	}
+
+	re := regexp.MustCompile(sslPairIdExpression)
+	if !re.MatchString(stringValue) {
+		return sslPairId, errors.New("InvalidSslPairId")
+	}
+
+	return SslPairId(stringValue), nil
+}
+
+func NewSslPairIdFromSslPairContent(
+	sslCertificate SslCertificateContent,
+	sslChainCertificates []SslCertificateContent,
+	sslPrivateKey SslPrivateKey,
+) (sslPairId SslPairId, err error) {
+	sslChainCertificatesMerged := ""
+	for _, sslChainCertificate := range sslChainCertificates {
+		sslChainCertificatesMerged += sslChainCertificate.String() + "\n"
+	}
+	contentToEncode := sslCertificate.String() + "\n" + sslChainCertificatesMerged + "\n" + sslPrivateKey.String()
+
+	sslPairIdContent, err := voHelper.TransformPlainContentIntoStrongHash(contentToEncode)
```

Comment:
Ahá, PlainContent means nothing indeed. It's just a string. The helper should be called StrongStringHasher or something like that. No need to create mystery on the code.